.clj-kondo/config.edn

```diff
@@ -1,9 +1,18 @@
-{:linters {:unresolved-symbol {:exclude [goog.DEBUG 
-                                         goog.string.unescapeEntities]}}
+{:linters
+ {:unresolved-symbol {:exclude [goog.DEBUG
+                                goog.string.unescapeEntities
+                                ;; TODO:lint: Fix when fixing all type hints
+                                object]}
+  ;; TODO:lint: Remove node-path excludes once we have a cleaner api
+  :unresolved-var {:exclude [frontend.util/node-path.basename
```

Comment:
These vars are not standard for clojurescript as we're exposing a js object and then calling with `.`. I'd recommend we either call these fns with `((gobj/get util/node-path "basename"))` or create fns for each js fn e.g. `(util/basename)`. I'm leaning towards the latter so we can change the implementation later if needed

Reply:
I prefer the latter too.

Reply:
Great! Will do